src/index.ts

```diff
@@ -202,7 +202,11 @@ class ClientApp {
             if ('gcHostOriginQueryParam' in cfg || 'gcTargetEnvQueryParam' in cfg) {
                 this.assertNonEmptyString(cfg.gcHostOriginQueryParam, 'host origin query param name');
                 this.assertNonEmptyString(cfg.gcTargetEnvQueryParam, 'target env query param name');
-                this._pcEnv = this.lookupGcEnv(parsedQueryString[cfg.gcHostOriginQueryParam], parsedQueryString[cfg.gcTargetEnvQueryParam]);
+                const parsedGcHostOrigin = parsedQueryString[cfg.gcHostOriginQueryParam];
+                const parsedGcTargetEnv = parsedQueryString[cfg.gcTargetEnvQueryParam];
+                this.assertNonEmptyString(parsedGcHostOrigin, 'host origin parsed query param');
+                this.assertNonEmptyString(parsedGcTargetEnv, 'target env parsed query param');
```

Comment:
where you able to find any changelog for query-string?  I dont see anything in the repo.  Since this involves a major version bump, it has potential to break.

was this failing at runtime or did you identify it as a breaking change?  fortunately for us, this does look like our only usage in the lib.  So, if we confirm this is still working correctly, we should be fine.

Reply:
I also was unable to find a changelog, but you are right that this is our only use for this package and I've tested it to make sure it still works. Although, one of the changes was them switching to typescript, which is how I caught this. Previously, parsedQueryString was type any, now its type is queryString.ParsedQuery<string>, so object keys are enforced as strings.